test/test.js

```diff
@@ -49,6 +51,14 @@ describe('compiler', () => {
     }
   })
 
+  test('accepts RegExps from other contexts', () => {
```

Comment:
This is clever, but I'm curious what your use case was for this :stuck_out_tongue:

Reply:
It probably will never come up, but I've run into it with other libraries, and it's usually better to be permissive with your is-it-a-builtin tests.